.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,10 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
+**What this PR does / why we need it**:
 
-1. If this is your first time sending a pull request, please read our contributing guidelines: https://github.com/kubermatic/kubeone/blob/master/CONTRIBUTING.md
-2. Make sure *all* commits in a pull request have the DCO signoff message. Without a DCO signoff, we can't review and merge your pull request due to legal reasons. Check the contributing guidelines for more information about DCO and how to sign commits: https://github.com/kubermatic/kubeone/blob/master/CONTRIBUTING.md#certificate-of-origin
-3. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
--->
```

Comment:
I think this is actually useful if you're just a contributor, we should consider leaving it.

Reply:
Okay, I added it back